### DISCUSSION TITLE
Server

### DISCUSSION THREAD

**Discussion by xbal023** [RioChndr/jaksel-language#55] [Q&A]
Bagaimana cara jaksel berjalan di sisi server?

**Comment by RioChndr** [comment]
Boleh di detailkan lagi maksud server-nya seperti apa ? jalan di CLI ? atau memberikan response berupa HTTP ? From: bolaxdSent: Monday, 17 October 2022 13:17To: RioChndr/jaksel-languageCc: SubscribedSubject: [RioChndr/jaksel-language] Server (Discussion #55) Bagaimana cara jaksel berjalan di sisi server?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Reply by xbal023** [reply]
Kalo gitu buat kan support require pak, yang bisa import javascript / typescript :)

**Comment by xbal023** [comment]
iya maksud saya seperti memberikan response ke HTTP, jadi nanti bisa membuat cloningan express dalam bahasa jaksel

**Comment by RioChndr** [comment]
Untuk itu belum support secara full dari jaksel script. Sent from Mail for Windows From: bolaxdSent: Monday, 17 October 2022 14:24To: RioChndr/jaksel-languageCc: Rio Chandra; CommentSubject: Re: [RioChndr/jaksel-language] Server (Discussion #55) iya maksud saya seperti memberikan response ke HTTP, jadi nanti bisa membuat cloningan express dalam bahasa jaksel—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
